chalicelib_fourfront/app_utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+from foursight_core.app_utils import app  # Chalice object
+from foursight_core.app_utils import AppUtils as AppUtils_from_core
+from foursight_core.identity import apply_identity_globally
+from .vars import FOURSIGHT_PREFIX, HOST
+
+
+class AppUtils(AppUtils_from_core):
+
+    # dmichaels/C4-826: Apply identity globally.
+    apply_identity_globally()
+
+    # Overridden from subclass.
+    APP_PACKAGE_NAME = "foursight"
+
+    # Note that this is set in the new (as of August 2022) apply_identity code;
+    # see foursight-core/foursight_core/{app_utils.py,identity.py}.
+    es_host = os.environ.get("ES_HOST")
+    if not es_host:
+        raise Exception("Foursight ES_HOST environment variable not set!")
+    HOST = es_host
+    
+    # overwriting parent class
+    prefix = FOURSIGHT_PREFIX
+    FAVICON = 'https://cgap-dbmi.hms.harvard.edu/static/img/favicon-fs.ico'
```

Comment:
This favicon should come from `data`, not CGAP.

Reply:
I see that in the current (non-React) version, for Fourfront-CGAP it comes from https://cgap-dbmi.hms.harvard.edu/static/img/favicon-fs.ico ... not sure what you mean exactly. For Foursight-Fourfront it is: https://data.4dnucleome.org/static/img/favicon-fs.ico

Reply:
You're in foursight-fourfront here, so you want https://data.4dnucleome.org/static/img/favicon-fs.ico

Reply:
Oh, right, of course, was looking at wrong one, thanks.